src/main/java/net/porillo/objects/OffsetBounty.java

```diff
@@ -3,14 +3,32 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import net.milkbowl.vault.economy.Economy;
+import net.milkbowl.vault.economy.EconomyResponse;
+import net.porillo.GlobalWarming;
+import net.porillo.config.Lang;
+import net.porillo.database.queries.insert.OffsetInsertQuery;
+import net.porillo.database.queue.AsyncDBQueue;
+import net.porillo.database.tables.OffsetTable;
+import net.porillo.database.tables.PlayerTable;
+import net.porillo.engine.ClimateEngine;
+import net.porillo.util.ChatTable;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
 
 import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 public class OffsetBounty {
 
+	private static final int MAX_BOUNTIES_CREATED_PER_PLAYER = GlobalWarming.getInstance().getConf().getMaxBounties();
```

Comment:
Should be careful declaring configurable variables as final constants, because we should still support /gw reload and I dont think this value would change properly after a reload but I could be wrong.

Reply:
I will have to try that (I was only aware of an op "/reload" -- which should work)

Reply:
I can confirm that "/reload" will pull the latest config values. "/gw reload" doesn't appear to be a registered command.